.circleci/config.yml

```
@@ -47,18 +55,24 @@ jobs:
       - when:
           condition: <<pipeline.parameters.run_e2e_tests>>
           steps:
-          
+          - run:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)